services/motion/builtin/builtin_test.go

```diff
@@ -412,10 +414,17 @@ func TestObstacleReplanning(t *testing.T) {
 
 	extra := map[string]interface{}{"max_replans": 0, "max_ik_solutions": 1, "smooth_iter": 1}
 
+	i := 0
```

Comment:
this is here so that when we first construct the plan and query the vision service for detection, we do not return anything. This way when we start executing the plan, the obstacle then appears.